neoq.go

```diff
@@ -22,7 +14,7 @@ import (
 // Neoq interface is Neoq's primary API
 type Neoq interface {
 	// Enqueue queues jobs to be executed asynchronously
-	Enqueue(ctx context.Context, job jobs.Job) (jobID int64, err error)
+	Enqueue(ctx context.Context, job *jobs.Job) (jobID int64, err error)
```

Comment:
`job` is a pretty big object. Pass it by reference.

Reply:
Yeah, and with arbitrary payload sizes it would mean a lot of copying.